src/main/kotlin/org/opensearch/commons/notifications/action/SendNotificationResponse.kt

```diff
@@ -33,38 +38,60 @@ class SendNotificationResponse : BaseResponse {
         @JvmStatic
         @Throws(IOException::class)
         fun parse(parser: XContentParser): SendNotificationResponse {
-            return SendNotificationResponse(NotificationEvent.parse(parser))
+            var notificationEvents: List<NotificationEvent>? = null
+
+            XContentParserUtils.ensureExpectedToken(
+                XContentParser.Token.START_OBJECT,
+                parser.currentToken(),
+                parser
+            )
+            while (parser.nextToken() != XContentParser.Token.END_OBJECT) {
+                val fieldName = parser.currentName()
+                parser.nextToken()
+                when (fieldName) {
+                    EVENT_LIST_TAG -> notificationEvents = parser.objectList { NotificationEvent.parse(it) }
+                    else -> {
+                        parser.skipChildren()
+                        log.info("Unexpected field: $fieldName, while parsing SendNotificationResponse")
```

Comment:
A warn log would be better here

Reply:
Kept it as `info` to maintain consistency across the code base. I can change it if you think otherwise.